cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
Is this an auto-generated file as well? If so, it should be added to the gitignore.

Also, I noticed running a test that fails creates a `cypress/screenshots` directory, which should be added to the gitignore as well.

Reply:
Yes, it is auto-generated file and I will add that too to gitignore